File: lib/model/account.dart
class Account {
  int? id;
  String phonenumber;
  String password;

  Account({
    this.id,
    required this.phonenumber,
    required this.password,
  });

  Map<String, dynamic> toMap() {
    var map = <String, dynamic>{
      'phonenumber': phonenumber,
      'password': password,
    };
    if (id != null) {
      map['id'] = id;
    }
    return map;
  }

  static Account fromMap(Map<String, dynamic> map) {
    return Account(
      id: map['id'],
      phonenumber: map['phonenumber'],
      password: map['password'],
    );
  }
}

File: backend/src/main/java/com/quanlychiteunhom/backend/entities/Account.java
package com.quanlychiteunhom.backend.entities;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "account")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Account {

    @Column(name = "id")
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "phonenumber")
    private String phonenumber;

    @Column(name = "password")
    private String password;

}
